src/CoreBundle/private/library/classes/dbobjects/TCMSTPLModule.class.php

```diff
@@ -210,7 +210,13 @@ public function getViewMapperConfig()
         if (null !== $viewMapperConfig) {
             return $viewMapperConfig;
         }
-        $viewMapperConfig = new ViewMapperConfig($this->sqlData['view_mapper_config']);
+
+        try {
+            $viewMapperConfig = $this->sqlData['view_mapper_config'];
```

Comment:
What kind of Exceptions can happen here?

a \array_key_exists would be sufficient?

Reply:
It is false by default (if not initialized). I changed the check to handle bool and missing array key here.